src/buddy/main.cpp

```diff
@@ -330,6 +314,19 @@ void spi_set_prescaler(SPI_HandleTypeDef *hspi, int prescaler_num) {
     HAL_SPI_Init(hspi);
 }
 
+void init_redscreen() {
+    // init lcd spi and timer for buzzer
+#if HAS_GUI()
+    SPI_INIT(lcd);
+    hw_tim2_init(); // TIM2 is used to generate buzzer PWM. Not needed without display.
+#endif
+
+    if (option::has_gui) {
```

Comment:
Use one condition type (Not `HAS_GUI()` and `if (option::has_gui)` next to it) and encapsule calling of the function